src/services/push-notifications.ts

```diff
@@ -13,18 +13,30 @@ const { PushNotifications } = Plugins
 export class PushNotificationsService {
     static createDefaultChannel() {
         const notificationChannel: NotificationChannel = {
-            id: 'visio-call-notifications', // id must match android/app/src/main/res/values/strings.xml's default_notification_channel_id
-            name: 'Visio call notifications',
-            description: 'Visio call notifications',
+            id: 'visio-call-notifications',
```

Comment:
Same as above

Reply:
I've changed the default_channel_id for the android manifest which remove any specifics of the default channel and is not used in the app